zookeeper/scripts/zookeeper_run.sh

```diff
@@ -54,5 +54,9 @@ echo "Starting Zookeeper with configuration:"
 cat /tmp/zookeeper.properties
 echo ""
 
+if [ -z "$ZOO_LOG4J_PROP" ]; then
```

Comment:
Similar to the Kafka Connect comment - do we want expose the variable as `ZOOKEEPER_LOG4J_OPTS` and map it only internally to `ZOO_LOG4J_PROP`?

Reply:
Ditto as above